pyroute2.ipset/pr2modules/wiset.py

```diff
@@ -251,7 +251,9 @@ def update_dict_content(self, ndmsg):
                         proto = IP_PROTOCOLS.get(proto, str(proto)).lower()
                         key += '{proto}:'.format(proto=proto)
                     key += str(entry.get_attr("IPSET_ATTR_PORT_FROM"))
-                key += ","
+                elif parse_type == "mac":
+                    key += entry.get_attr("IPSET_ATTR_ETHER")
+               key += ","
```

Comment:
Hello,

Good catch. But could you submit your patch again without this weird indentation change on the last line? Thanks

Reply:
Sorry for the typo - I've pushed a fix commit.